_posts/2019-10-3-ashlyn.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: Go Hacktoberfest
```

Comment:
Please change your `title` to something unique, as the current one clashes with [this](https://github.com/sunwaytechclub/hacktoberfest-2019/blob/master/_posts/2019-10-3-Go-Hacktoberfest-2019.md).